src/widgetastic_patternfly4/bulletchart.py

```diff
@@ -0,0 +1,161 @@
+import re
+
+from cached_property import cached_property
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Text
+from widgetastic.widget import Widget
+from widgetastic.xpath import quote
+
+
+class Legend:
+    """Represents Legend of chart."""
+
+    def __init__(self, label, value=None, color=None, element=None):
+        self.label = label
+        self.value = value
+        self.color = color
+        self.element = element
+
+    def click(self):
+        self.element.click()
+
+    def __repr__(self):
+        return f"Legend({self.label}: {self.value})"
+
+    def __gt__(self, leg):
+        return self.__class__ == leg.__class__ and self.value > leg.value
+
+    def __eq__(self, leg):
+        return (
+            self.__class__ == self.__class__ and self.label == leg.label and self.value == leg.value
+        )
+
+
+class Bar(Legend):
+    """Represents Bar of chart."""
+
+    def __repr__(self):
+        return f"Bar({self.label}: {self.value})"
+
+
+class BulletChart(Widget):
```

Comment:
It seems `BulletChart` contains other widgets. Do you think it might be better to convert it to view?

Reply:
agreed :+1: